src/components/my-history/custom-my-history.vue

```diff
@@ -20,8 +20,24 @@
           >
             <HistoryIcon class="max-desktop:x-icon-lg" />
 
-            <div class="x-flex x-flex-col x-gap-2">
-              <p>{{ suggestion.query }}</p>
+            <div class="x-group x-flex x-flex-col x-gap-2">
+              <p class="group-hover:x-underline">{{ suggestion.query }}</p>
+              <div
+                v-if="suggestion.selectedFilters && suggestion.selectedFilters.length"
+                class="x-text1-sm x-flex x-gap-8 x-text-lead-50"
+              >
+                <div class="x-line-clamp-1">
```

Comment:
You're still using line clamp instead of truncate.
Also, this should have a width of 320px aprox.